pkg/kubectl/describe.go

```diff
@@ -162,6 +162,25 @@ func init() {
 	DefaultObjectDescriber = d
 }
 
+func getStorageClassAnnotation(obj api.ObjectMeta) string {
+	if obj.Annotations[storage.BetaStorageClassAnnotation] != "" {
```

Comment:
I think this would be better as below so correctly handly the explicitly set to empty.
```
    if class, ok := obj.Annotations[storage.BetaStorageClassAnnotation]; ok {
        return class
    }
```